salt/redash.sls

```diff
@@ -0,0 +1,6 @@
+{% from 'lib.sls' import create_pg_database %}
+
+include:
+  - docker_apps
+
+{{ create_pg_database( "redash_db", "redash_user" ) }}
```

Comment:
@jpmckinney I see the `create_pg_database` macro has been removed, how is this now configured?

Reply:
I've modified this to use the new system in `salt/postgres/init.sls` / putting the databases in pillar.
Might need some tweaking, I couldn't find any documentation or use of this new set up yet.

Reply:
It's step 2 here. I'll fix the heading for the section to make it clearer.

Reply:
https://ocdsdeploy.readthedocs.io/en/latest/develop/update/postgres.html#add-service-accounts

Reply:
Happy the PR is updated for new postgres install + configuration process.